docker/Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM ubuntu:15.04
+# This base image is equivalent to Ubuntu 16.04 but has better defaults
+# and fixes numerous problems with the stock Ubuntu images.
+# read more about it here: https://github.com/phusion/baseimage-docker
+FROM phusion/baseimage:0.9.19
```

Comment:
Without a compelling reason to use `phusion/baseimage`, I would prefer to use the official `ubuntu:16.04`.